protocols/gossipsub/src/generated/gossipsub/pb.rs

```diff
@@ -9,6 +9,7 @@
 #![cfg_attr(rustfmt, rustfmt_skip)]
 
 
+use bytes::Bytes;
```

Comment:
These files are auto-generated, you can't change them.

Reply:
Ah sorry! I didn't realise this was generated (but makes sense).  I'll update the PR to avoid these changes.

I'm not super familiar w/ protobuf, is it something that could be re-generated to utilise `Bytes`, or this is the interface we have to deal with?

Reply:
We use https://github.com/tafia/quick-protobuf which does not support `Bytes` but maybe it can be contributed? :)